### DISCUSSION TITLE
Small theme changes

### DISCUSSION THREAD

**Discussion by arcanis** [facebook/docusaurus#6162] [Q&A]
It seems the classic Docusaurus theme got slightly more ... cramped, I think (note how the text is closer from the edges, and sidebar items take a little bit less vertical space)? Could there be an easy way to revert these changes in our themes, potentially via overridable variables?

Before:
<img src="https://user-images.githubusercontent.com/1037931/147088386-1f293ccb-5ea8-43f6-be47-4d5d5baac4c6.png" width="400"/>

After:
<img src="https://user-images.githubusercontent.com/1037931/147088356-5cac4991-550f-4f1b-b8c3-39e32aa389db.png" width="400"/>

**Comment by johnnyreilly** [comment]
Gosh your eyes are good! I barely noticed the changes 😂

**Comment by slorber** [comment]
Actually, this is one of the goals of v2.0 to make this kind of customization more stable and robust: https://github.com/facebook/docusaurus/issues/6113

We don't really have a public API surface for such customizations. Adding a CSS var to control spacing everywhere would probably not work great in all cases (it would probably add some spacing in places you don't want to, or we would need to create many CSS variables, and are these CSS variables public API surface anyway? 🤷‍♂️ )

Adding stable class names to many theme components should help you achieve this with custom CSS.

Maybe tell us exactly what are the elements you would want to customize or target with CSS, and we can allow making these customizations easily.
